install/pre-install-payload/Makefile

```diff
@@ -1,11 +1,13 @@
 COCO_CONTAINERD_VERSION = 1.6.8.2
 OFFICIAL_CONTAINERD_VERSION = 1.7.0
 VFIO_GPU_CONTAINERD_VERSION = 1.7.0.0
+NYDUS_SNAPSHOTTER_VERSION = v0.13.2
```

Comment:
Hi @fidencio,

v0.13.2 contains the fixes? Because on CCv0 we are still on v0.12.0 (https://github.com/kata-containers/kata-containers/blob/CCv0/versions.yaml#L305)

Reply:
We're just using the latest here, and that's a known to be working version.
We shouldn't be lacking behind because of CCv0.

Reply:
Agree. I'd like to double-check though.